### DISCUSSION TITLE
Coloring the flow buttons with the flow selected color

### DISCUSSION THREAD

**Discussion by Mehdi-YC** [directus/directus#22539] [Feature Requests]
### Summary

changing the color of the flow button based on the flow color selected in the setup

### Basic Example

Hello Directus,
this is a freature that will make it more easy to make the difference between flows
![image](https://github.com/directus/directus/assets/44881216/c8e2a80d-5eff-44e8-b633-c6c4dbaef14a)
here i have 2 flows , they both have same gb color, and it's more interesting to have the color selected on the flow setup

### Motivation

the benefit of this is to easily make the difference between the flows buttons
so better UI/UX

### Detailed Design

we already select a color when we create a flow,
and i expected it to be used for the buttons on the flow section,
and it is not
so i am making this feature request
and i will be happy to implement it myself

### Must Have:
- coloring the button with the same flow selected color

### Drawbacks

there is no drawback, and the feature is simple to implement

### Alternatives

using custom css for every button,
it is harder to maintain, and also for now we don't do anything with the selected color

### Adoption Strategy

this will break nothing as it only change the appearance of the buttons

**Comment by MarkBurvs** [comment]
Yes please! Honestly it surprised me that it didn't work like this already when I started digging into Flows.

**Reply by Mehdi-YC** [reply]
Well, if you can add some upvotes, we might be able to get this soon, i hope so, i can also work on it

**Comment by Borty97** [comment]
Hey! That’s a good idea. Voted!

**Comment by rijkvanzanten** [comment]
@Mehdi-YC Wanna open a PR for this? 🙂 It's a bit tricky for me to prioritize this, but feels like a great addition

**Reply by Mehdi-YC** [reply]
I will try to do it during this week , it's been a while since i took a look into directus code , so this will take some time, but i'll work on it

**Reply by Mehdi-YC** [reply]
Hello @rijkvanzanten,

i created a PR for this one https://github.com/directus/directus/pull/25192

**Comment by Mehdi-YC** [comment]
Hi @Borty97 and @MarkBurvs,
the PR is accepted, it might be available in the next directus release

**Reply by MarkBurvs** [reply]
Amazing. Thanks all!
